src/pages/screen-2.tsx

```diff
@@ -77,7 +80,50 @@ export default class Screen2 extends React.Component {
       this.state.userWEP = false;
       let userWEP = false;
     }
-    var userPension = Number(SessionStore.get("pensionAmount"))
+    
+    // //////////////////////////////////////
+    
+    // let dob = dayjs(dayjs(SessionStore.get("BirthDate")).format(), dayjs.ISO_8601)
+    // console.log('dob-incoming', userDOB)
+    // console.log('dob', dob)
+    
+    let dob = dayjs(userDOB)
+    let now = dayjs()
+    let workerAge = now.diff(dob, "year")
+    let userPension
+    // let ageAwarded = 
+
+    console.log('dob', dob)
+    console.log('now', now)
+    console.log('workerAge', workerAge)
+    
+    // //////////////////////////////////////
+
+    if (SessionStore.get("coveredEmployment")) {
+      if (SessionStore.get("pensionOrRetirementAccount") === "MONTHLYPENSION") {
+        console.log('monthly')
+        userPension = Number(SessionStore.get("pensionAmount"))
+      } else if (SessionStore.get("pensionOrRetirementAccount") === "LUMPSUMRETIREMENTACCOUNT") {
+        console.log('lump')
+        userPension = ObsFuncs.lumpSumToMonthly(
+          SessionStore.get("pensionAmount"),
+          dayjs(new Date(JSON.parse(SessionStore.get("dateAwarded"))).toLocaleDateString("en-US")),
+          workerAge,
+          dob
+        )
+        console.log('userPension', userPension)
+      } else {
+        console.log('other')
+        userPension = Number("0")
```

Comment:
We should avoid setting pension to zero when it is really `undefined`. In the [upcoming version](https://github.com/codeforboston/windfall-elimination/pull/130) I have a placeholder in that field that turns red when it is actually `undefined` so users aren't caught off guard when users start typing and to make it clear they have to type something if they say they have pension/account:
<img width="698" alt="image" src="https://user-images.githubusercontent.com/283343/70292709-e579cd00-17ac-11ea-911b-325b79b89416.png">

Reply:
Would it be better to simply remove that row?

Reply:
I changed it to undefined, and it works